extensions/wikia/UserProfilePageV3/css/UserProfilePage.scss

```diff
@@ -148,6 +148,20 @@ $color-user-profile-gradient-bottom: desaturate(darken($color-menu-highlight, 5%
 				margin: 0 1px 0 6px;
 			}
 		}
+	  	#discussion-bio {
+		  max-height: 100px;
+		  overflow: hidden;
+		  transition: max-height 1s;
```

Comment:
The transition is no longer being used now that the overflow text is put in a newly created modal upon expanding. Looks like I missed this when cleaning up earlier.